lib/sourcemap.js

```diff
@@ -44,12 +44,11 @@
 "use strict";
 
 import MOZ_SourceMap from "source-map";
```

Comment:
We still use the `SourceMapGenerator` API to generate a new sourcemap for the minified output. `TraceMap` (and `AnyMap`) are only for consuming and tracing an already existing sourcemap.